blockchain/blockchain.go

```diff
@@ -96,6 +96,12 @@ type Blockchain interface {
 	BlockFooterByHash(h hash.Hash256) (*block.Footer, error)
 	// GetTotalActions returns the total number of actions
 	GetTotalActions() (uint64, error)
+	// GetNumActions returns the number of actions
+	GetNumActions(height uint64) (uint64, error)
+	// GetTranferAmount returns the transfer amount
+	GetTranferAmount(height uint64) (*big.Int, error)
+	// CalculateTransferAmount returns the calculated transfer amount
+	CalculateTransferAmount(blkActions []action.SealedEnvelope) *big.Int
```

Comment:
This doesn't need to be part of blockchain API. It seems to be an utility method, and doesn't access any blockchain resource. Consider move it to an util.go if we want to avoid duplicate CalculateTransferAmount usage in api.go and blockdao.go